contracts/HashRegistrarSimplified.sol

```diff
@@ -238,7 +242,38 @@ contract Registrar {
         }
         return len;
     }
+    
+    /** 
+     * @dev Creates a scheduled release for each name
+     * 
+     * Each name will be assigned a random date in which its auction 
+     * can be started, from 0 to 6 months
+     * 
+     * @param _hash The hash to start an auction on
+     * @param _timestamp The timestamp to query about
+     */
+     
+    function isAllowed(bytes32 _hash, uint _timestamp) constant returns (bool allowed){
```

Comment:
I think it'd be a lot cleaner if this function just called `getAllowedTime` and compare it to the current time.